code/src/main/java/com/onlinecontacttracing/messaging/filters/NumberOfMessagesFlaggingFilter.java

```diff
@@ -0,0 +1,20 @@
+package com.onlinecontacttracing.messaging.filters;
+
+import com.onlinecontacttracing.messaging.filters.FlaggingFilter;
+import com.onlinecontacttracing.storage.PositiveUser;
+import com.onlinecontacttracing.storage.PotentialContact;
+import java.lang.Exception;
+
+public class NumberOfMessagesFlaggingFilter implements FlaggingFilter{
+  private static int limitNumOfMessages = 100;
+
+  public boolean passesFilter(PositiveUser positiveUser, String message) {
+    if (positiveUser.getNumberOfEmailsSent() > limitNumOfMessages) {
```

Comment:
If you have a guard that just returns a boolean, you can just return the expression itself.

So, 

return !positiveUser.getNumberOfEmailsSent() > limitNumOfMesssages;